src/eval/execute.rs

```diff
@@ -178,6 +202,20 @@ fn generate_result_from_response(resp: Response, channel: Channel, is_private: b
     }
 }
 
+/// result will be an url with an additional '\n'
+async fn paste_to_pb(client: &Client, code: &str) -> reqwest::Result<String> {
+    const DPASTE_API_URL: &str = "https://paste.mozilla.org/api/";
+
+    let resp = client
+        .post(DPASTE_API_URL)
+        .form(&[("lexer", "rust"), ("content", code), ("format", "url")])
+        .send()
+        .await?
+        .text()
```

Comment:
Have you resolved the API issue?

Reply:
yeah
Content-Length is really useless
I got the missing Content Length error with an ill formed request

Reply:
Nah, I'm talking about the response format. Can it return the paste URL without the quotes now?

Reply:
indeed
with `format` set as `url` it will simply return ".../..\n"